impls/powershell/Dockerfile

```diff
@@ -21,16 +19,10 @@ WORKDIR /mal
 # Specific implementation requirements
 ##########################################################
 
-# Nothing additional needed for python
-RUN apt-get -y install libunwind8 libicu52
-#RUN apt-get -y install libunwind8 libicu55
-
-# For dist packaging
-RUN curl -L -O https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.9/powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb && \
-    dpkg -i powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb && \
-    rm powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb
-#RUN curl -L -O https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.9/powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb && \
-#    dpkg -i powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb && \
-#    rm powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
+RUN apt-get-y install ca-certificates curl
```

Comment:
Looks like missing space ("apt-get-y" -> "apt-get -y") causing the image build to fail. I experimented a bit trying to build the image with the updated powershell. Seems like the powershell deb doesn't quite support 24.04 yet (due to dep on libicu that doesn't include the version in 24.04). Forcing the install seems to allow powershell to install and run:
```
diff --git a/impls/powershell/Dockerfile b/impls/powershell/Dockerfile
index e7e957e5..365881a6 100644
--- a/impls/powershell/Dockerfile
+++ b/impls/powershell/Dockerfile
@@ -19,10 +19,10 @@ WORKDIR /mal
 # Specific implementation requirements
 ##########################################################

-RUN apt-get-y install ca-certificates curl
+RUN apt-get -y install ca-certificates curl libicu74
 RUN curl -L -O \
     https://github.com/PowerShell/PowerShell/releases/download/v7.4.5/powershell_7.4.5-1.deb_amd64.deb && \
-    dpkg -i powershell_7.4.5-1.deb_amd64.deb && \
+    dpkg -i --force-depends powershell_7.4.5-1.deb_amd64.deb && \
     rm powershell_7.4.5-1.deb_amd64.deb
```
I think using Ubuntu 22.04 and libicu72 also seems to work.

In either case, when I try and use that to test the current state of this branch, it seems to fail in step1 when processing a "+" sign:
```
user> +

An error has occurred that was not properly handled. Additional information is shown below. The PowerShell process will exit.
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at CallSite.Target(Closure, CallSite, Object, ExecutionContext)
```

Reverting to the original version of the image that has powershell 6.0.0-alpha.9, all tests pass with your code changes. You have to use the original upstream docker image (because vivid image can't be rebuilt any more).
```
make impl_to_image=ghcr.io/kanaka/mal-test-powershell:20200211_055016-g8a19f603 DOCKERIZE=1 test^powershell
```

Looks like Ubuntu 16.04 packages are still available, so the following would allow the image to be rebuilt and seems to still pass all the tests:
```
diff --git a/impls/powershell/Dockerfile b/impls/powershell/Dockerfile
index e7e957e5..1ecddb9d 100644
--- a/impls/powershell/Dockerfile
+++ b/impls/powershell/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:24.04
+FROM ubuntu:16.04
 MAINTAINER Joel Martin <github@martintribe.org>

 ##########################################################
@@ -9,8 +9,10 @@ MAINTAINER Joel Martin <github@martintribe.org>
 RUN apt-get -y update

 # Required for running tests
-RUN apt-get -y install make python3
-RUN ln -fs /usr/bin/python3 /usr/local/bin/python
+RUN apt-get -y install make python
+
+# Some typical implementation and test requirements
+RUN apt-get -y install curl libreadline-dev libedit-dev

 RUN mkdir -p /mal
 WORKDIR /mal
@@ -19,10 +21,12 @@ WORKDIR /mal
 # Specific implementation requirements
 ##########################################################

-RUN apt-get-y install ca-certificates curl
-RUN curl -L -O \
-    https://github.com/PowerShell/PowerShell/releases/download/v7.4.5/powershell_7.4.5-1.deb_amd64.deb && \
-    dpkg -i powershell_7.4.5-1.deb_amd64.deb && \
-    rm powershell_7.4.5-1.deb_amd64.deb
+# Nothing additional needed for python
+RUN apt-get -y install libunwind8 libicu55
+
+# For dist packaging
+RUN curl -L -O https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.9/powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb && \
+    dpkg -i powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb && \
+    rm powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb

 ENV HOME=/mal
 ```